src/commit/orb.yml

```diff
@@ -0,0 +1,39 @@
+version: 2.1
+ 
+commands:
+  version-tag: 
+    parameters:
+      format:
+        default: 'number-tag'
+        description: |
+          Create a more human readable version tag for your builds. Place this step after checkout on
+          your job and it will create an environment variable $VERSION_TAG to use in later steps.
+          Often used to tag docker images or create github release.
+
+          Options are ['number-tag', 'tag-only', 'number-only']
+          'number-tag: 152-afed12g'
+          'tag-only: afed12g'
+          'number-only: 152'
+        type: enum
+        enum: ['number-tag', 'tag-only', 'number-only']
+    example:
+    steps:
+      - run:
+          name: Create Version Tag
+          command: |
+            echo 'export COMMIT_NUM=$(git rev-list --count $CIRCLE_SHA1)' >> $BASH_ENV
```

Comment:
Why not compute all these values right now and save the actual tag into a var in `$BASH_ENV`? That way if something goes wrong you would see it in this step, rather in the next step where `$BASH_ENV` is parsed.

Reply:
Yeah, great idea. I'll add that tomorrow.